### ISSUE TITLE
Training steps

### ISSUE THREAD

**Issue by Malaika68** [khdlr/HED-UNet#2]
Hey can you please help me with the training steps? how can I use my dataset and its labels?

**Comment by khdlr** [comment]
Hi Malaika,

you'll need to implement a [`torch.utils.data.Dataset`](https://pytorch.org/docs/stable/data.html#torch.utils.data.Dataset) that returns pairs of image and ground truth annotations.

Then you can change the `get_dataset` function in `data_loading.py` to use your custom dataset instead of the pre-configured InriaDataset.

Let me know if that helps!

Cheers,
Konrad

**Comment by Malaika68** [comment]
Thank you so much for the help. Can you please tell what type of annotations it should be?

**Comment by khdlr** [comment]
No problem! The annotations should be of the same shape as the imagery (i.e. dense annotations) and contain a 1 for each foreground pixel and a 0 for each background pixel.

In the examples shown in the README, Glacier/Building pixels are the "foreground"

**Comment by Malaika68** [comment]
Okay great,,
